pkg/apis/velero/v1/backup_repository_types.go

```diff
@@ -51,6 +51,9 @@ const (
 	BackupRepositoryPhaseNew      BackupRepositoryPhase = "New"
 	BackupRepositoryPhaseReady    BackupRepositoryPhase = "Ready"
 	BackupRepositoryPhaseNotReady BackupRepositoryPhase = "NotReady"
+
+	BackupRepositoryTypeRestic string = "restic"
+	BackupRepositoryTypeKopia  string = "kopia"
```

Comment:
Please change this variable name, and change its to something like "udmrepo" or "unified-repo".
We don't expose Kopia as a repository directly, it is a provider of unified repo and is managed by it.